index.css

```diff
@@ -0,0 +1,696 @@
+:root {
+  --white: #fff;
+  --black: #1c2b2d;
+  --blue: #31326f;
+  --light-blue: #005490;
+  --color-primary: #9d0191;
+  --color-sec: #db6400;
+  --color-grey: #eee;
+  --color-dark-grey: #222831;
+  --heading-font-family: 'Cocogoose', sans-serif;
+  --main-font-family: 'Lato', sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  font-family: 'Lato', sans-serif;
+}
+
+.begin {
+  background-image: url('images/mobile-background.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-bottom: 100px;
+}
+
+header {
+  height: 100px;
+  width: 100%;
+  background: #2e2e2e;
+  position: fixed;
+  right: 0.2px;
+  z-index: 10;
+}
```

Comment:
- [ ] Nice work implementing the hamburger menu on the mobile version :clap: However it would be nice to make its background transparent or have the same color/pattern as the top section background to follow the provided design
### Your design :point_down: 
![Screenshot from 2023-07-06 10-28-11](https://github.com/Zuwairanajma/ModuleOne-Capstone-Project/assets/91142432/74c770b8-8f6d-4f62-b2da-628494b32e97)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-07-06 10-27-51](https://github.com/Zuwairanajma/ModuleOne-Capstone-Project/assets/91142432/9d75aae4-e292-4b3d-b256-b14cc60b7161)